toc/README.md

```diff
@@ -0,0 +1,88 @@
+- The nominee must have a deep understanding of TiDB or other projects in TiDB ecosystem.
+- The nominee must belong to a certain organization of TOC membership and be able to participate in TOC on behalf of his/her organization.
+- The nominee must put the community first and be able to balance the relationship between the interests of the organization and the interests of the community.
+
+
+## Vote for election
+
+After nomination, the existing TOC members will vote for election and decide by [lazy consensus](./Proposal/voting-rules.md#lazy-consensus).
+
+## Members
+
+The current list of TOC members is as below.
+
+|  Name   | GitHub  | Represents | Membership |
+|  ----  | ----  |  ----  | ----  |
+| Xiaoguang Sun sunxiaoguang@pingcap.com  | [sunxiaoguang](https://github.com/sunxiaoguang) | PingCAP | TOC Chairman |
+| Siddon Tang tl@pingcap.com | [siddontang](https://github.com/siddontang) | PingCAP | TOC Official Member |
+| Chengxuan Xu bigpyer@gmail.com | [niubell](https://github.com/niubell) | PalFish | TOC Official Member |
+| Yuqing Bai baiyuqing@zhihu.com | [baiyuqing](https://github.com/baiyuqing) | Zhihu | TOC Official Member |
+| Jian Zhang zhangjian@pingcap.com | [zz-jason](https://github.com/zz-jason) | TiDB | TOC Official Member |
+| Jinpeng Zhang zhangjinpeng@pingcap.com | [zhangjinpeng1987](https://github.com/zhangjinpeng1987) | TiKV | TOC Official Member |
+| Chengwen Yin cwen@pingcap.com | [cwen0](https://github.com/cwen0) | Chaos Mesh®  | TOC Official Member |
+| Dian Luo luodian@pingcap.com | [andylokandy](https://github.com/andylokandy) | TiKV | TOC Non-voting Member |
+| Jay Li jay@pingcap.com | [BusyJay](https://github.com/BusyJay) | TiKV | TOC Non-voting Member |
+| Cong Wang wangcong@pingcap.com | [bb7133](https://github.com/bb7133) | TiDB | TOC Non-voting Member |
+
+
+### Emeritus Members
+
+TOC members has no concept of tenure, but will retire under the following circumstances:
+
+- Actively choose to retire due to personal reasons.
+- When a TOC member can no longer participate in TOC affairs on behalf of this organization due to leaving or withdrawing from the organization they represent.
+- The organization the TOC member represents no longer exists.
+
+Retirees will become emeritus members of TOC. The list of emeritus members is as below.
+
+- Xianlin Chen，Head of Technology Hub at PalFish
+- Qiang Zhou，Chaos Mesh® Maintainer
+
+
+## TOC Meeting
+The TOC monthly online meeting is held at 8 pm (UTC+08:00) on the last Friday of each month. Meeting invitations will be sent through the TOC mail group: https://lists.tidb.io/g/toc. 
+
+If official members cannot attend the monthly meeting for special reasons, they must ask for leave in advance.
+
+### Meeting Minutes
+
+[English version](https://docs.google.com/document/d/1m9Uk-JkqO_KaSZaLjL47hW1L_2Bp-yW1StCuGQzOFTk/edit)
+
+[Chinese version](https://docs.google.com/document/d/1KvavvwWpABbTEJO3IMLBykGMOUMq4soL2ityDryDqgw/edit)
```

Comment:
IMHO, just maintain English meeting minus is enough for TOC?

Reply:
OK, I will delete the Chinese Version.